images/build/distroless-iptables/distroless/Dockerfile

```diff
@@ -24,6 +24,7 @@ COPY package-utils.sh /
 # TODO: /bin/sleep is used by a specific e2e test
 ARG STAGE_DIR="/opt/stage"
 RUN apt -y update && \
+    apt -y dist-upgrade && \
```

Comment:
Should not be this done in the base image?

Reply:
I mean, why does this happen?

Reply:
Good question - this happens because there is a delay between when the new library is made available in stable, e.g.:

- https://tracker.debian.org/news/1432726/accepted-openssl-111n-0deb11u5-source-into-stable-security/

And when a new base image is actually updated / released in the dockerhub repo:

- https://github.com/docker-library/official-images/blob/master/library/debian

Reply:
Updated the description to reflect that.